target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -313,6 +313,12 @@ rp-n53)
 	ucidef_set_led_netdev "eth" "Network" "$board:white:back" "eth0"
 	set_wifi_led "$board:blue:wifi"
 	;;
+rt-ac51u)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.1
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.2
+	ucidef_set_wifi_led "$board:blue:activity"
```

Comment:
Please remove the ```ucidef_set_wifi_led``` here. The board has a dedicated 2.4/5GHz LED which should be used. If the "activity" led is indeed the 2.4/5GHz LED, the LED should be renamed in the dts and the usb LED needs to be added.